connect/src/com/telenor/connect/ConnectSdk.java

```diff
@@ -329,16 +329,15 @@ private static void sendAnalyticsData() {
         sendAnalyticsData(null);
     }
 
-    public static void sendAnalyticsData(Throwable e) {
-        WellKnownAPI.WellKnownConfig wellKnownConfig = getWellKnownConfig();
-        if (wellKnownConfig == null) {
-            return;
-        }
-        String analyticsEndpoint = wellKnownConfig.getAnalyticsEndpoint();
-        if (analyticsEndpoint == null) {
+    public static void sendAnalyticsData(Throwable throwable) {
+        String analyticsEndpoint;
+        // This is bad practise. The source of the issue must be found.
```

Comment:
Typo. Please change practise to practice :)

Reply:
@metatelenor Valid point, will fix that.